CONTRIBUTING.md

```diff
@@ -1,5 +1,36 @@
+
+Submit a pull-request
+---------------------
 I actively welcome your pull requests.
 
 1. Fork the repo and create your branch from `develop`.
 2. If you've added code that should be tested, add tests.
 3. New pull request to `develop`.
+
+How to report a bug
```

Comment:
I would also like to explain how to leave an issue on this page.
But the issue template and format I'm using seems to be different. It would be better to unify it with a better bug reporting template.
[Check My Issue Template](https://github.com/sangcomz/FishBun/blob/master/.github/ISSUE_TEMPLATE/bug_report.md)

Reply:
got it! I'll add a link for the templates